sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryTracer.java

```diff
@@ -189,14 +193,50 @@ public Context getSharedSpanBuilder(String spanName, Context context) {
         return context.addData(SPAN_BUILDER_KEY, getSpanBuilder(spanName, context));
     }
 
+    @Override
+    public void addEvent(String eventName, Map<String, Object> attributes, Instant timestamp) {
+        Span currentSpan = Span.current();
+        if (currentSpan == null) {
+            logger.info("Failed to find a starting span to associate an event with.");
+            return;
+        }
+
+        currentSpan.addEvent(
+            eventName,
+            attributes == null ? Attributes.empty() : convertToAttributes(attributes),
+            timestamp
```

Comment:
Can eventName or timestamp be `null`?

Reply:
eventName cannot be. Timestamp could be bull. Updated javadoc to include that.